### DISCUSSION TITLE
Is the Pre-Release nightly available on Bookworm ?

### DISCUSSION THREAD

**Discussion by fitaine** [merbanan/rtl_433#2530] [Q&A] [answered]
Hi all,

My **Orium 13123** weather station is not fully supported by the **22.11** version of rtl_433 so I need to **update to the Pre-Release nightly** to add support [(2300)](https://github.com/merbanan/rtl_433/pull/2300). Is this possible and may you tell me how ?

I use a raspberry with Raspberry OS Bookworm and made the install with CMake.

Thank you !

**Comment by zuckschwerdt** [comment, marked as answer]
You will need to compile from source yourself, see https://github.com/merbanan/rtl_433/blob/master/docs/BUILDING.md

**Reply by fitaine** [reply]
ok, thanks I was able to to it after some research.
